statedb/treeview.go

```diff
@@ -42,9 +44,11 @@ type TreeViewer interface {
 	// Iterate iterates over all leafs of this tree.  When callback returns true,
 	// the iteration is stopped and this function returns.
 	Iterate(callback func(key, value []byte) bool) error
-	// Dump exports all the tree leafs in a byte array.
-	Dump() ([]byte, error)
-	// Root returns the root of the tree, which cryptographically summarises the
+	// Dump exports all the tree leafs into a writer buffer.
+	Dump(w io.Writer) error
+	// Import writes a buffer containing all the tree leafs.
```

Comment:
Can you rephrase it? I'm quite bad with words :/

Reply:
I thought it was a typo: import takes a reader, so it feels wrong that it says it "writes". Perhaps something like this, but tweak it because I'm assuming how it works internally:

> Import reads exported tree leaves from r.